lib/ansible/config/base.yml

```diff
@@ -190,6 +190,15 @@ BECOME_ALLOW_SAME_USER:
   - {key: become_allow_same_user, section: privilege_escalation}
   type: boolean
   yaml: {key: privilege_escalation.become_allow_same_user}
+BECOME_PASSWORD_FILE:
+  name: Become password file
+  default: ~
+  description: 'The password file to use for the become plugin. --become-password-file.'
+  env: [{name: ANSIBLE_CONNECTION_PASSWORD_FILE}]
```

Comment:
Is this supposed to be `ANSIBLE_BECOME_PASSWORD_FILE`?

Reply:
yep